src/sipp.cpp

```diff
@@ -2271,10 +2267,4 @@ int main(int argc, char *argv[])
     close(epollfd);
     free(epollevents);
 #endif
-
-    if (scenario_file != NULL) {
-        delete [] scenario_file;
-        scenario_file = NULL;
-    }
-
```

Comment:
I'm not sure what you guys think of this. IMHO, its pointless to waste time here, the OS will just reclaim that memory anyways...

Unless we care about valgrind, but we can always provide a suppression file.